README.md

```diff
@@ -30,10 +30,21 @@ running tail.
 
 ## Development
 
+Install your local copy:
+
 ```bash
 $ pip install -e .
 ```
 
+Run tests:
+
+```bash
+$ rm -rf tests/build && pip install -t tests/build .
+$ python -m unittest
+```
+
+If weird things happen, it might be worth running `find . -name __pycache__ | xargs rm -rf`
```

Comment:
Did this ever happen again? I'd vote for either elaborating on "weird stuff" or getting rid of the comment. I honestly don't even remember what happened anymore.

Reply:
It didn't, and I don't remember exactly _what_ happened. I believe after we did this that we also ran `pip install -t tests/build .`.